src/main/java/trade/wayruha/mexc/dto/MexcWSResponse.java

```diff
@@ -0,0 +1,38 @@
+package trade.wayruha.mexc.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+import static java.util.Objects.isNull;
+
+@Data
+public class MexcWSResponse<T> {
+
+    //Properties for base response from stream
+    private String id;
+    private String code;
+    @JsonProperty("msg")
+    private String message;
+
+    //Main properties
+    @JsonProperty("d")
+    private T data;
+    @JsonProperty("c")
+    private String channel;
+    @JsonProperty("s")
+    private String symbol;
+    @JsonProperty("t")
+    private Long timestamp;
+
+    public String toStringShort() {
+        return "MexcWSResponse_Short{" +
+                "id='" + id + '\'' +
```

Comment:
this symbol (\') only makes toString looks worse (more dirty), doesn't it? what is the purpose?